### PR TITLE
Add DI favicon

### PR DESCRIPTION
**Pull request by dalepotter**
Adds the favicon at http://devinit.org/wp-content/themes/diTheme/img/siteIcon.png

image

Tested (and working) in Chrome 62.0.3202.94 (Official Build) (64-bit) and Firefox 57.0 (64-bit).

~Live preview: http://17b9b313.ngrok.io/dashboard~ < No longer active

### CODE REVIEW

**Review by PetyaKangalova — APPROVED** [review]

**Review by hayfield — APPROVED** [review]
:paintbrush:

**Comment by wendyrogers** [comment]
This change is marked as complete but doesn't appear to have been implemented? Is it still awaiting deployment somewhere?

**Comment by dalepotter** [comment]
@wendyrogers whilst this was merged into the `dev` branch at the time, it was subsequently grouped in to a separate branch (`phase1-integration-branch`) to include all phase one changes that continue to await GB governance approval.

As per the note in https://github.com/devinit/grand-bargain-monitoring/pull/79 once all changes are approved, it will be merged to dev and deployed.

**Pull request merged**
